error: unreachable expression
  --> $DIR/expr_call.rs:22:17
   |
22 |     foo(return, 22); //~ ERROR unreachable
   |                 ^^
   |
note: lint level defined here
  --> $DIR/expr_call.rs:14:9
   |
14 | #![deny(unreachable_code)]
   |         ^^^^^^^^^^^^^^^^

error: unreachable expression
  --> $DIR/expr_call.rs:27:5
   |
27 |     bar(return); //~ ERROR unreachable
   |     ^^^^^^^^^^^

error: aborting due to 2 previous errors

