error: unreachable expression
  --> $DIR/expr_array.rs:19:34
   |
19 |     let x: [usize; 2] = [return, 22]; //~ ERROR unreachable
   |                                  ^^
   |
note: lint level defined here
  --> $DIR/expr_array.rs:14:9
   |
14 | #![deny(unreachable_code)]
   |         ^^^^^^^^^^^^^^^^

error: unreachable expression
  --> $DIR/expr_array.rs:24:25
   |
24 |     let x: [usize; 2] = [22, return]; //~ ERROR unreachable
   |                         ^^^^^^^^^^^^

error: aborting due to 2 previous errors

