error: unreachable expression
  --> $DIR/expr_method.rs:25:21
   |
25 |     Foo.foo(return, 22); //~ ERROR unreachable
   |                     ^^
   |
note: lint level defined here
  --> $DIR/expr_method.rs:14:9
   |
14 | #![deny(unreachable_code)]
   |         ^^^^^^^^^^^^^^^^

error: unreachable expression
  --> $DIR/expr_method.rs:30:5
   |
30 |     Foo.bar(return); //~ ERROR unreachable
   |     ^^^^^^^^^^^^^^^

error: aborting due to 2 previous errors

