error: unreachable expression
  --> $DIR/expr_assign.rs:19:5
   |
19 |     x = return; //~ ERROR unreachable
   |     ^^^^^^^^^^
   |
note: lint level defined here
  --> $DIR/expr_assign.rs:14:9
   |
14 | #![deny(unreachable_code)]
   |         ^^^^^^^^^^^^^^^^

error: unreachable expression
  --> $DIR/expr_assign.rs:29:14
   |
29 |         *p = return; //~ ERROR unreachable
   |              ^^^^^^

error: unreachable expression
  --> $DIR/expr_assign.rs:35:15
   |
35 |     *{return; &mut i} = 22; //~ ERROR unreachable
   |               ^^^^^^

error: aborting due to 3 previous errors

